pyproject.toml

```diff
@@ -17,22 +17,15 @@ warn_unused_configs = true
 show_error_codes = true
 show_column_numbers = true
 ignore_missing_imports = true
+# NOTE: Do not grow the exclude list. Edit .lintrunner.toml instead
 exclude = [
-    '\.git',
-    '__pycache__',
-    '^build',
     '^third_party',
-    '_pb2.py$',
-    '\.cache',
-    '\.eggs',
-    '\.setuptools-cmake-build',
-    'venv/',
```

Comment:
Is it ok to remove excluding these files here?

Reply:
Yes, because lintrunner knows to skip anything not tracked by git as well as excluded files in the lintrunner config